cli/dcoscli/cluster/main.py

```diff
@@ -308,21 +313,26 @@ def _unlink(name):
     """
     Unlink a DC/OS cluster.
 
-    :param name:  name of the cluster
+    :param name: ID or name of the cluster
     :type name: str
     :returns: process status
     :rtype: int
     """
 
     c = cluster.get_cluster(name)
-    if not c:
-        raise DCOSException('Unknown cluster {}.'.format(name))
+    if c:
+        name = c.get_cluster_id()
 
     dcos_url = config.get_config_val('core.dcos_url')
     endpoint = urllib.parse.urljoin(
-            dcos_url, '/cluster/v1/links/' + c.get_cluster_id())
+            dcos_url, '/cluster/v1/links/' + name)
 
-    http.delete(endpoint)
+    try:
+        http.delete(endpoint)
+    except DCOSHTTPException as e:
+        if e.status() == 404:
+            raise DCOSException('Unknown cluster link {}.'.format(name))
```

Comment:
Would something like `"Cluster '{}' could not be found".format(name)` be better here?

Reply:
I wanted to add the `link` semantic to make it clear what it is about, if we adapt this message we should probably also align the other one with it. Currently it is : 
- `This cluster link already exists.`
- `Unknown cluster link <id_or_name>.`

cc @pavisandhu @klueska